elisp/ghc-info.el

```diff
@@ -31,10 +31,25 @@
 ;;; type
 ;;;
 
+(defface ghc-type-region
```

Comment:
This face makes text under it quite unreadable, at least with my theme. How about we use an underline face or something less intrusive instead? Or maybe base the default colors for this theme on ones from the default faces of the current theme and modify them a bit.